packages/app-desktop/gui/NoteEditor/NoteBody/TinyMCE/TinyMCE.tsx

```diff
@@ -1041,7 +1043,13 @@ const TinyMCE = (props: NoteBodyEditorProps, ref: any) => {
 		async function onCopy(event: any) {
 			const copiedContent = editor.selection.getContent();
 			const copyableContent = getCopyableContent(copiedContent);
-			clipboard.writeHTML(copyableContent);
+			if (copyableContent.startsWith('<img')) {
+				const imagePath = getImagePath(copyableContent);
+				const image = nativeImage.createFromPath(imagePath);
+				clipboard.writeImage(image);
+			} else {
+				clipboard.writeHTML(copyableContent);
+			}
```

Comment:
We need something more generic than this. For example, if I make a selection that includes some text and an image, it will still not be possible to paste it in Google Docs. Most likely the image needs to be converted to a format understood by Google Docs, perhaps as a data URL.

Reply:
> We need something more generic than this. For example, if I make a selection that includes some text and an image, it will still not be possible to paste it in Google Docs. 

Yes, I also thought the same regarding multiple selection thing. But as the issue was regarding copying a single image using Ctrl+C, I tried to solve the issue confining that only.

Reply:
Yes, but I don't think there's any shortcut for this feature. Trying to handle only one image means things like `copyableContent.startsWith('<img')` which will work in only a very narrow case. I'm sure that 90% of the time there will be some extra text or spaces before or after the selection and the solution will break.

Reply:
> Trying to handle only one image means things like `copyableContent.startsWith('<img')` which will work in only a very narrow case. 

Sorry, but I am a bit confused. The issue specifically said that a **single image** should be copied when the image it is selected and copy command is executed through keyboard(Ctrl + C).  

> I'm sure that 90% of the time there will be some extra text or spaces before or after the selection and the solution will break.

Thus I think selecting a single image and then copying will not break the solution, as selection of an image guarantees that nothing other than it is selected.(not even text)

Nevertheless, If you don't agree with the solution, please tell the what should I add / improvise. I will do it.
Thanks :)

Reply:
You should be able to select some text and images in the rich text editor, paste it into Google Doc, and it should work. Currently, with your implementation, that would not work.